Ponca/src/Fitting/algebraicSphere.h

```diff
@@ -249,7 +249,7 @@ class AlgebraicSphere : public T
     PONCA_MULTIARCH inline VectorType primitiveGradient (const VectorType& _q) const;
 
     /*! \brief Approximation of the scalar field gradient at the evaluation point */
```

Comment:
```suggestion
    /*! \brief Approximation of the scalar field gradient at the evaluation point */
        \warning The gradient is not normalized by default
```

But, if we consider the polynomial $u_c + u_l x + u_q x x$, the gradient is supposed to be $u_l + 2u_q x$, right  ?

Reply:
Yes, this is what is computed in `primitiveGradient (const VectorType& _q)` with a position as parameter `_q`.